app/models/cart.rb

```diff
@@ -0,0 +1,8 @@
+class Cart < ActiveRecord::Base
+  belongs_to :user
+  has_many :line_items
+
+  def total_cart_price
+    line_items.to_a.sum(&:total_price)
```

Comment:
This isn't an issue in this case so there's nothing to change, but be aware that `sum` will raise an error if the array has anything that it can't add in the array (including nil). Whenever I see `sum` on something like this without filtering the the array, a little bell goes off so I thought I should mention it. So if there was any chance that the `LineItem#total_price` method will return nil, there would be cases that your application would throw a 500 from this method.